common/web/input-processor/README.md

```diff
@@ -8,14 +8,8 @@ The Keyboard Processor module is an internal component of KeymanWeb, seen within
 * A local installation of [Node.js](https://nodejs.org/) v8.9+.
 	* Builds will call `npm install` to automatically install further necessary build dependencies.
 
-	* Linux users can run the following to update to LTS version of nodejs
-	
-```
-sudo apt-get install python-software-properties
-curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
-sudo apt-get install nodejs
-```
+	* Linux users can update to LTS version of nodejs by following instructions on the [NodeSource Distributions](https://github.com/nodesource/distributions#table-of-contents) page.
```

Comment:
I'd be inclined just to leave this out as the details are in docs/build/linux-ubuntu.md anyway. Less to maintain.

Reply:
so just a pointer? how much of this doc should be removed? there are a couple like this.

Reply:
without this change, users following the existing instructions will get the giant "THIS SCRIPT IS GOING AWAY" message.  maybe we should move forward with this and make another task for reorganizing the docs?